### ISSUE TITLE
[Bug Report]: Package Swiftmailer "abandoned"

### ISSUE THREAD

**Issue by EmTeeAge** [BookStackApp/BookStack#3123]
### Describe the Bug

Hi!
Sorry to bother, if it's a nonsense issue, feel free to close it :)

I recently tried updating from v21.08.3 to v21.12 and noticed this information when running `composer install --no-dev`

```
...
  - Installing league/oauth2-client (2.6.0): Extracting archive
  - Installing phpseclib/phpseclib (3.0.12): Extracting archive
  - Upgrading predis/predis (v1.1.7 => v1.1.9): Extracting archive
  - Upgrading socialiteproviders/microsoft-azure (4.2.1 => 5.0.1): Extracting archive
Package swiftmailer/swiftmailer is abandoned, you should avoid using it. Use symfony/mailer instead.
Generating optimized autoload files
composer/package-versions-deprecated: Generating version class...
composer/package-versions-deprecated: ...done generating version class
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi
...
```
In the output, there's a mention of an abandoned package. `Package swiftmailer/swiftmailer is abandoned, you should avoid using it. Use symfony/mailer instead.` When I checked the [GitHub repo of this package](https://github.com/swiftmailer/swiftmailer), it is indeed archived and says this in the Readme:

> Swiftmailer will stop being maintained at the end of November 2021.
> 
> Please, move to Symfony Mailer at your earliest convenience. Symfony Mailer is the next evolution of Swiftmailer. It provides the same features with support for modern PHP code and support for third-party providers.

My question is - is this package used in BookStack? If so, is it considered a problem or are there any plans to move to symfony/mailer?

I've tried looking through the BookStack's GitHub and blog, but haven't found a mention of this, but as I've mentioned - feel free to close this issue if it doesn't make sense.

Thank you.

### Steps to Reproduce

Run `composer install --no-dev during` the update process.

### Expected Behaviour

Command should finish without warnings or errors.

### Exact BookStack Version

v21.08.3 while updating to v21.12

### PHP Version

PHP 7.4.25 (cli)

### Hosting Environment

Debian 11 5.10.0-10-amd64 VPS

**Comment by ssddanbrown** [comment]
Thanks for reporting @EmTeeAge.

This is a known thing. Swiftmailer is used within BookStack via our framework, Laravel. 
This is something that's going to be [changed in the next version of Laravel](https://github.com/laravel/framework/pull/38481) but it may be a year or so before we're able to move onto that therefore this message may show for a while upon upgrades.

While Swiftmailer is abandoned it's fairly stable/mature package that should hopefully not require active maintenance until we can upgrade although I'll be keeping an eye out for any issues we may need to patch in the meantime.

I'll keep this open for now though for visibility of others searching the same thing.

**Comment by ssddanbrown** [comment]
This has now been addressed as part of #4021, and will therefore not be a problem as of our next feature release.
